src/site/content/en/blog/optimize-fid/index.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Optimize First Input Delay
+subhead: How to respond faster to user interactions.
+authors:
+  - houssein
+  - addyosmani
+date: 2020-05-05
+hero: hero.png
+alt: Optimize FID banner
+description: | 
+  First Input Delay (FID) measures the time from when a user first interacts with your site
+  to the time when the browser is actually able to respond to that interaction. Learn how to
+  optimize FID by minimizing unused JavaScript, breaking up Long Tasks, and improving interaction
+  readiness.
+tags:
+  - post
+  - fast
+  - web-vitals
+  - first-input-delay
+  - fid
+  - lighthouse
+---
+
+<blockquote>
+  <p>
+    I clicked but nothing happened! Why can't I interact with this page? 😢
+  </p>
+</blockquote>
+
+[First Contentful Paint](/fcp/) (FCP) and [Largest Contentful
+Paint](/lcp/) (LCP) are both metrics that measure the time it takes for content to
+visually render (paint) on a page. Although important, paint times do not capture _load
+responsiveness_: or how quickly a page responds to user interaction.
+
+[First Input Delay](/fid/) (FID) is a [Core Web Vitals](/metrics/) metric that captures a user's
+first impression of a site's interactivity and responsiveness. It measures the time from when a user
+first interacts with a page to the time when the browser is actually able to respond to that
+interaction. FID is a [field metric](/user-centric-performance-metrics/#in-the-field) and cannot be
+simulated in a lab environment. **A real user interaction** is required in order to measure the
+response delay.
+
+<picture>
+  <source srcset="../vitals/fid_8x2.svg" media="(min-width: 640px)">
+  <img class="w-screenshot w-screenshot--filled"
+      src="../vitals/fid_4x3.svg"
+      alt="Good fid values are 2.5 seconds, poor values are greater than 4.0
+            seconds and anything in between needs improvement">
+</picture>
+
+To approximate FID in the [lab](/how-to-measure-speed/#lab-data-vs-field-data), we
+recommend [Total Blocking Time (TBT)](/tbt/). They measure different things, but
+improvements in TBT usually correspond to improvements in FID. 
+
+The main cause of a poor FID is **heavy JavaScript execution**. Optimizing how JavaScript parses,
+compiles, and executes on your web page will directly reduce input delay.
+
+## Heavy JavaScript execution
+
+The browser cannot respond to user input while it's executing JavaScript. In other words, the
+browser can't respond to user interactions while the main thread is busy. To improve this:
+
++   [Reduce JavaScript execution time](#reduce-javascript-execution)
++   [Break up Long Tasks](#break-long-tasks)
++   [Optimize your page for interaction readiness](#optimize-interaction-readiness)
++   [Use a web worker](#web-worker)
+
+### Reduce JavaScript execution time {: #reduce-javascript-execution }
+
+Reducing the amount of JavaScript on your page reduces the amount of time that the browser needs to
```

Comment:
You have "reduce" twice. It might flow better if you say either:

- "Reducing the amount of JavaScript on your page **lessens** the amount..."
- "**Limiting** the amount of JavaScript on your page reduces the amount..."

Reply:
```suggestion
Limiting the amount of JavaScript on your page reduces the amount of time that the browser needs to
```